modules/sdk-core/src/bitgo/lightning/lightning.ts

```diff
@@ -98,4 +100,18 @@ export class Lightning implements ILightning {
       throw new Error(`error(s) parsing response body: ${errors}`);
     });
   }
+
+  public async getInvoices(query?: GetInvoicesQuery): Promise<GetInvoicesResponse> {
+    const queryParams: { status?: string; limit?: number } = {};
+    queryParams.status = query?.status;
+    queryParams.limit = query?.limit;
```

Comment:
Nit but I think this is a tiny bit cleaner:

```suggestion
    const queryParams = {
      status: query?.status,
      limit: query?.limit
    }
```
```